tests/test_onnx2py.py

```diff
@@ -5,14 +5,13 @@
 import onnxruntime as _ort
 import sys
 
-from onnxconverter_common.onnx2py import convert, clear_directory
-
 working_path = os.path.abspath(os.path.dirname(__file__))
 tmp_path = os.path.join(working_path, 'temp')
 data_path = os.path.join(working_path, 'data')
 
 class Oonnx2PyTests(unittest.TestCase):
     def tearDown(self):
+        from onnxconverter_common.onnx2py import clear_directory
```

Comment:
wasn't sure if this is a good practice in general, any special reason of moving it here?

Reply:
onnx2py requires onnx version > 1.3.  The new `from onnx import external_data_helper` will make it crash when imported in lower versions.  The python 5 unit tests use onnx 1.3 but are skipped for onnx2py.

Reply:
Ideally maybe onnx2py would be its own repo since its pip requirements are technically higher than onnxconverter-common's install requirements.